README.md

```diff
@@ -31,35 +31,58 @@ The ```main``` branch of this repository contains samples configured for the lat
 
 The *ArcGIS Runtime SDK Samples app* has a *Target SDK* version of *13.0*, meaning that it can run on devices with *iOS 13.0* or newer.
 
-## Building samples using installed SDK
+## Building Samples Using Swift Package Manager
 
 1. **Fork** and then **clone** the repository
-    > Make sure to use the "recursive" option to ensure you get the **ArcGIS Runtime Toolkit** submodule
-    >
-    >`git clone --recursive [URL to Git repo]`
-    >
-    > If you've already cloned the repo without the submodule, you can load the submodule using 
-    >
-    >`git submodule update --init`
+1. **Run** the `arcgis-ios-sdk-samples` app target
+
+> The project is configured to install the ArcGIS Runtime SDK for iOS and ArcGIS Runtime Toolkit with Swift Package Manager.
+> To add the Swift Packages to your project, follow the steps below
+
+* Open your project in Xcode
+* Go to File > Swift Packages > Add Package Dependency option
+* Enter `https://github.com/Esri/arcgis-runtime-toolkit-ios` as the package repository URL
+* Choose version 100.11.0 or a later version. Click Next. Only version 100.11.0 or newer supports Swift Package Manager.
+
+> Note: The Toolkit Swift Package adds the ArcGIS Runtime SDK Swift Package as a dependency, so there is no need to add both separately.
+> 
+> New to Swift Package Manager? Visit [here](https://swift.org/package-manager/)
+
+## Building Samples Using Manually Installed SDK
+
 1. **Install** the ArcGIS Runtime SDK for iOS to a central location on your Mac as described [here](https://developers.arcgis.com/ios/get-started)
+1. **Fork** and then **clone** the repository
+1. **Download** the package repository for ArcGIS Runtime SDK for iOS [here](https://github.com/Esri/arcgis-runtime-ios), and extract it to `$HOME/arcgis-runtime-ios`
+1. **Create** a symbolic link to the ArcGIS Runtime SDK binary framework
+    > `ln -s $HOME/Library/SDKs/ArcGIS/Frameworks/ArcGIS.xcframework $HOME/arcgis-runtime-ios/ArcGIS.xcframework`
+1. **Open** the package manifest (Package.swift) for ArcGIS Runtime SDK
+    > Change target to
+    > `.binaryTarget(name: "ArcGIS", path: "ArcGIS.xcframework")`
+1. **Download** the package repository for ArcGIS Runtime Toolkit for iOS [here](https://github.com/Esri/arcgis-runtime-toolkit-ios), and extract it to the project directory
+1. **Drag** the Toolkit's folder into the Project navigator
+    > This action adds your dependency’s Swift package as a local package to your project.
+1. **Open** the Toolkit's package manifest
+    > Add `import FileProvider`
+    >
+    > Change dependency to
+    > `.package(name: "ArcGIS", path: "\(FileManager.default.homeDirectoryForCurrentUser.path)/arcgis-runtime-ios")`
```

Comment:
@yo1995 Thank you for figuring this out. But for now, let's exclude this section for the upcoming release. I'm a little uncomfortable on how unwieldy it is. We can focus only focus on SPM and cocoapods for the time being while we have some more discussion on the manual option.

Reply:
Agreed 👍

Reply:
I also agree. Basically if we still want manual support in the future, maybe we need to somehow include 2 package manifests in this repo, and/or have a script to download the source code for toolkit and set things up, so that the whole project's codebase can go completely offline.